include/bluetooth/services/bms.rst

```diff
@@ -0,0 +1,42 @@
+.. _bms_readme:
+
+GATT Bond Management Service (BMS)
+##################################
+
+This module implements the Bond Management Service with the corresponding set of characteristics defined in the BMS specification.
```

Comment:
"BMS specification" should link to https://www.bluetooth.com/specifications/gatt/

Reply:
fixed